src/main/kotlin/app/revanced/cli/command/utility/InstallCommand.kt

```diff
@@ -44,20 +45,35 @@ internal object InstallCommand : Runnable {
                 }.install(Installer.Apk(apk, packageName))
             } catch (e: Exception) {
                 logger.severe(e.toString())
+                throw e
             }
 
             when (result) {
-                RootInstallerResult.FAILURE ->
+                RootInstallerResult.SUCCESS ->
+                    logger.info("Mounted the APK file")
+                RootInstallerResult.FAILURE -> {
                     logger.severe("Failed to mount the APK file")
-                is AdbInstallerResult.Failure ->
-                    logger.severe(result.exception.toString())
-                else ->
+                    throw Exception()
+                }
+                AdbInstallerResult.Success ->
                     logger.info("Installed the APK file")
+                is AdbInstallerResult.Failure -> {
+                    logger.severe(result.exception.toString())
+                    throw Exception()
+                }
+                else -> {
+                    logger.severe("Unknown installation result")
```

Comment:
This case will never occur.

Reply:
At the moment, true. I prefer to leave these in for covering my future-self ass, in case a new condition is introduced that's not handled.

Happy to remove if you insist.

Reply:
This makes sense. Exhausting the cases is reasonable in case new implementations of the abstract classes in library is added